src/ripple/protocol/Feature.h

```diff
@@ -355,6 +355,7 @@ extern uint256 const fixFillOrKill;
 extern uint256 const fixNFTokenReserve;
 extern uint256 const fixInnerObjTemplate;
 extern uint256 const featurePriceOracle;
+extern uint256 const fixNFTokenTrustlineSurprise;
```

Comment:
That's a cute name, but it seems a little to jocular. Maybe `fixEnforceNFTokenTrustline`?